apps/mobile/src/components/ContextSwitchMenu/ContextSwitchMenu.js

```diff
@@ -33,10 +33,13 @@ export default function ContextSwitchMenu ({ isExpanded, setIsExpanded, fullView
   // Sort pinned by navOrder, unpinned by group name
   const sortedPinnedGroups = pinnedMemberships
     .sort((a, b) => a.navOrder - b.navOrder)
-    .map(m => GroupPresenter(m.group))
+    .map(m => m.group ? GroupPresenter(m.group) : null)
+    .filter(Boolean) // Remove null entries
```

Comment:
honestly i dont love having all these safety checks. yes it may prevent a crash but then it would lead to a more subtle bug we wouldnt catch, of groups just not appearing when they should